<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Param</class>
 <widget class="QDialog" name="Param">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>156</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>293</width>
    <height>156</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>293</width>
    <height>156</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Param</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Param√®tres</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEditTemps"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lNom">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Nom :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEditNom"/>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEditPoints"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="lPoints">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Max de points : </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lTemps">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>10</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Temps Max : </string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="bAnnuler">
          <property name="focusPolicy">
           <enum>Qt::NoFocus</enum>
          </property>
          <property name="text">
           <string>Annuler</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="bValider">
          <property name="text">
           <string>Valider</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditNom</tabstop>
  <tabstop>lineEditPoints</tabstop>
  <tabstop>lineEditTemps</tabstop>
  <tabstop>bAnnuler</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
